schemas/3DScenesConfiguration/v1.0.0/3DScenesConfiguration.schema.json

```diff
@@ -236,6 +236,30 @@
                 "time"
             ]
         },
+        "IADXTimeSeriesConnection": {
+            "type": "object",
+            "description": "Azure Data Explorer connection information for time series data",
+            "additionalProperties": false,
+            "properties": {
+                "adxClusterUrl": {
+                    "type": "string"
+                },
+                "adxDatabaseName": {
+                    "type": "string"
+                },
+                "adxTableName": {
+                    "type": "string"
+                }
+            },
+            "required": ["adxClusterUrl", "adxDatabaseName", "adxTableName"]
+        },
+        "ITimeSeriesConnection": {
+            "anyOf": [
```

Comment:
why do we need to use `anyof` here? can't we just directly tell it the type is of `IADXTimeSeriesConnection`? Presumably this is for extensibility?

Reply:
yes I mentioned in the description, it is for extensibility in the future for any other type of connection that we might support